### DISCUSSION TITLE
Using the Citations Plugin to connect Obsidian with Zotero

### DISCUSSION THREAD

**Discussion by MatBlab** [retorquere/zotero-better-bibtex#2824] [Q&A]
Hello everyone,

I've been using the Citations Plugin in Obsidian to creature literature notes of articles that I saved in Zotero. When I used a certain command, Obsidian would search my database in Zotero for the article I wanted. Recently, I noticed that Obsidian can't seem to find any of the newer articles I saved in Zotero and I have no idea why. I checked my Zotero add ons and the Obsidian plugins, but I can't find the problem. (I'm also very new at this.) When I save the articles in Zotero, they appear in Obsidian in a specific folder, so that seems to work.

If someone has any ideas, I would be very grateful.
MatBlab

**Comment by retorquere** [comment]
It would start with the obsidian side. They're fetching data, we're not pushing data, so if data is not showing up, it's because that side is net getting it.
